server/build_mobile_apps.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"github.com/mattermost/mattermost-mattermod/model"
+)
+
+func (s *Server) buildMobileApp(pr *model.PullRequest) {
```

Comment:
this will just create the new branch, how we know the pipeline for this new branch passed? and also i think we need to comment in the PR the links to download the built packages

Reply:
Do you mean, if the PR checks themselves have passed before trying to build the mobile package? I forgot about that. Makes totally sense to add it, good catch. Not yet sure how to get the link to the built packages. Maybe it could also be done in the circleci part of the repo instead and posted into a channel (that was @enahum's workaround suggestion). I will find a solution.

Reply:
currently it posts to the QA channel, but if we can construct them and post into the PR that would be AWESOME!